src/Microsoft.Health.Fhir.Core/Features/Persistence/Orchestration/BundleOrchestratorOperationResource.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+
+namespace Microsoft.Health.Fhir.Core.Features.Persistence.Orchestration
+{
+    public sealed class BundleOrchestratorOperationResource<T>
```

Comment:
Can this class be removed? It doesn't look like it is used.

Reply:
Thanks. It was on my original design, but I found ways to avoid using it.